### PR TITLE
NAS-121136 / 23.10 / Add "Select Pool" dialog mechanism to new Apps pages

### PR DESCRIPTION
**Pull request by AlexKarpov98**
Go to new Apps section and check the ticket details:
You should see.

Screenshot 2023-03-22 at 17 21 23

### CODE REVIEW

File: src/app/pages/apps/components/pool-configuration-checker/pool-configuration-checker.component.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { SelectPoolDialogComponent } from 'app/pages/apps/components/select-pool-dialog/select-pool-dialog.component';
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-pool-configuration-checker',
+  templateUrl: './pool-configuration-checker.component.html',
+  styleUrls: ['./pool-configuration-checker.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PoolConfigurationCheckerComponent implements AfterViewInit {
```
Comment (undsoft) [inline src/app/pages/apps/components/pool-configuration-checker/pool-configuration-checker.component.ts:16]: Why not just a service with some method that would get called onInit inside of available and installed apps?

File: src/app/pages/apps/components/available-apps/available-apps.component.ts [hunk 2]
```diff
@@ -130,4 +132,10 @@ export class AvailableAppsComponent implements OnInit, AfterViewInit {
 
     this.cdr.markForCheck();
   }
+
+  updateToolbarButtons(): void {
+    if (this.appsToolbarButtons) {
+      this.appsToolbarButtons.ngOnInit();
```
[the remark this answers is not in the document]
Comment (AlexKarpov98) [reply src/app/pages/apps/components/available-apps/available-apps.component.ts:138]: okay, then I can make a public method to call.

File: src/app/pages/apps/components/pool-configuration-checker/pool-configuration-checker.component.html
[the remark this answers is not in the document]
Comment (AlexKarpov98) [reply src/app/pages/apps/components/pool-configuration-checker/pool-configuration-checker.component.html:1]: oh, missed this one

**Comment by AlexKarpov98** [comment]
gonna be fixed another way using this ticket: 
https://ixsystems.atlassian.net/browse/NAS-121291

**Pull request closed without merging**